pom.xml

```diff
@@ -355,6 +355,20 @@ under the License.
                     </dependency>
                 </dependencies>
             </plugin>
+
+            <plugin>
+                <groupId>org.cyclonedx</groupId>
+                <artifactId>cyclonedx-maven-plugin</artifactId>
```

Comment:
?

Reply:
Yeah this shouldn't have been here. Removed